hera_cal/omni.py

```diff
@@ -1297,7 +1300,10 @@ def omni_apply(files, opts):
             for bl, k in zip(*np.unique(mir.baseline_array, return_index=True)):
                 blmask = np.where(mir.baseline_array == bl)[0]
                 ai, aj = mir.baseline_to_antnums(bl)
-                if not ai in cal.ant_array or not aj in cal.ant_array:
+                if ai not in cal.ant_array or aj not in cal.ant_array:
+                    if opts.flag_missing:
+                        # flag the visibilities if either antenna is missing from calibration solutions
+                        mir.flag_array[blmask, nsp, :, p] = True
```

Comment:
Des the flag array not need to be an array? Does the True transfer to all freqs and times?

Reply:
Yes, the `True` value broadcasts to all of the specified array indices